### ISSUE TITLE
list of files not refreshing when using mount

### ISSUE THREAD

**Issue by jope2000** [ncw/rclone#799]
Hi there,

I'm currently experimenting with rclone and the new mount functionality.
Rclone 1.33 is running in a docker environment (docker alljoysnville/rclone:latest).
The connection to google drive is working fine and I can upload/download file no problem.

The only problem I have is that when I add/remove files on my google drive file list doesn't change in my mounted directory. Only once I unmount and mount the google drive again, the file list will get updated.

here are the commands that I'm running:
`rclone mount gdrive:remoteDirectory path/to/local/ &`
`ls path/to/local` -> this will give me all the files are there.

I then copy a new file into the directory on google drive (e.g. using the web interface or another computer).
Then I look at the list of files in my docker container again. The newly copied file is missing from that list.

If I use `rclone ls gdrive:remoteDirectory` directly, all changes are reflected immediately.

Is there some kind of delay to get this file list updated?

**Comment by scriptzteam** [comment]
I made it myself for ACD, and working great! it is auto-unmoun and auto-mount every 5min. via cron. All you need is setup your paths ...

https://github.com/ncw/rclone/issues/790#issuecomment-253769752

**Comment by jope2000** [comment]
that might be an option if you are just dealing with small files.
However, what if you are in the middle of accessing a larger file and your cron re-mounts the drive?

**Comment by ncw** [comment]
I think this is a duplicate of #680 which I'm planning to fix soon! However @scriptzteam solution may work for you in the mean time.

**Comment by jope2000** [comment]
Yes that is the same problem that I'm seeing. just reproduced in a slightly different manner.
This can be marked as a duplicate.

**Comment by ncw** [comment]
@jope2000 thanks.  If you click the subscribe button on the other issue, you'll get a notification when it is updated (eg when I post a beta with a fix).

**Comment by scriptzteam** [comment]
"that might be an option if you are just dealing with small files.
However, what if you are in the middle of accessing a larger file and your cron re-mounts the drive?"

use flasfxp or some ftp client when accessing mounted folder, ftp support resume of failed download ;) so you will start downloading where you end, so not the whole file again ;)
